prov/net/src/xnet_progress.c

```diff
@@ -876,6 +994,8 @@ xnet_handle_events(struct xnet_progress *progress,
 	}
 
 	xnet_handle_event_list(progress);
+	xnet_submit_io_uring(&progress->tx_io_uring);
+	xnet_submit_io_uring(&progress->rx_io_uring);
```

Comment:
I think these are acting somewhat like a deferred doorbell notification.  I would move check for using io_uring here, and wrap both calls.

Reply:
Sure, will do.